.github/workflows/pr-title.yml

```diff
@@ -19,9 +20,6 @@ on:
   merge_group:
     types: [checks_requested]
 
-permissions:
-  pull-requests: write
-
```

Comment:
This was the problem that caused errors for callers that didn't specify write access.